core/src/main/scala/org/apache/spark/scheduler/TaskSetManager.scala

```diff
@@ -671,8 +671,10 @@ private[spark] class TaskSetManager(
           if (blacklistedEverywhere) {
             val partition = tasks(indexInTaskSet).partitionId
             abort(s"Aborting $taskSet because task $indexInTaskSet (partition $partition) " +
-              s"cannot run anywhere due to node and executor blacklist.  Blacklisting behavior " +
-              s"can be configured via spark.blacklist.*.")
+              s"cannot run anywhere due to node and executor blacklist.\n" +
```

Comment:
Can we change to Scala's triple quoted string interpolation here? you can refer to [here](https://github.com/apache/spark/blob/ceaec93839d18a20e0cd78b70f3ea71872dce0a4/core/src/main/scala/org/apache/spark/api/python/SerDeUtil.scala#L211)

Reply:
Nice @jerryshao i will update as soon as possible.